circuits/builder/rotate.rs

```diff
@@ -346,11 +375,12 @@ pub mod tests {
         // we can use the first 32 bytes of the header as the seed to get_fixed_subarray, but this
         // is not correct.
         let target_header_dummy_hash = &target_header.header_bytes.as_vec()[0..32];
-        let prefix_subarray = builder.get_fixed_subarray::<MAX_HEADER_SIZE, MAX_PREFIX_LENGTH>(
-            &header_as_variables,
-            start_position,
-            target_header_dummy_hash,
-        );
+        let prefix_subarray = builder
+            .get_fixed_subarray_unsafe::<MAX_HEADER_SIZE, MAX_PREFIX_LENGTH>(
```

Comment:
do you need to fix the usage here?

Reply:
In tests, it doesn't matter to pass the correct seed. Leaving this out for now